jekyll/_cci2/configuration-reference.md

```diff
@@ -743,6 +743,44 @@ version | Y | String | Should currently be `2`
 
 A unique name for your workflow.
 
+### **`triggers`**
+A workflow can have a `trigger` to run on a schedule.
+
+Key | Required | Type | Description
+----|-----------|------|------------
+triggers | N | String | Should currently be `schedule`.
+{: class="table table-striped"}
+
+#### **`type`**
+A workflow may have a `type` of `schedule` indicating it runs on a schedule, for example a nightly build that runs every day at 12am UTC:
+
+```
+workflows:
+   version: 2
+   nightly:
+     triggers:
+       - type: schedule
+         cron: "0 0 * * *"
+         branches:
+           - master
+           - beta
```

Comment:
@nwjsmith Should we add jobs here? Which jobs are going to get executed in this example?

Reply:
oh or the idea is _just_ to show the format?

Reply:
It would be useful to show a really simple `jobs` stanza, probably only need a single job. What about:

```
workflows:
  version: 2
  nightly:
    triggers:
      - type: schedule
        cron: "0 0 * * *"
        branches:
          - master
          - beta
    jobs:
      - test
```
?

Reply:
yup makes sense

Reply:
added, thanks!